src/services/ui/src/components/Inputs/textarea.tsx

```diff
@@ -3,19 +3,37 @@ import * as React from "react";
 import { cn } from "@/lib/utils";
 
 export interface TextareaProps
-  extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {}
+  extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {
+  charcount?: "simple" | "limited";
+  charcountstyling?: string;
```

Comment:
Could this be:
charcountStyling?

Reply:
Where it's being applied it causes a loud warning about needing to be lowercase for custom DOM elements.

There's probably a better way to add these two properties in general.

Reply:
I see makes sense